src/com/cinergix/mapper/ObjectMapper.java

```diff
@@ -33,14 +40,125 @@
 	 * @param dataClass Type of object in list which is returned by this method.
 	 * @return This method returns a list of object f type <code>T</code>
 	 */
-	public List<T> mapResultSetToObject( ResultSet result, Class<T> dataClass ) {
+public List<T> mapResultSetToObject( ResultSet result, Class<T> dataClass ) throws InstantiationException, IllegalAccessException, SQLException {
```

Comment:
Remove the irrelevant exceptions being thrown. Only throw exceptions related to the users expectation